File: src/test/java/cernx/colt/matrix/tfloat/impl/DenseFloatMatrix2DViewTest.java
package cernx.colt.matrix.tfloat.impl;

public class DenseFloatMatrix2DViewTest extends DenseFloatMatrix2DTest {

    public DenseFloatMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new DenseFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new DenseFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new DenseFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}

File: src/test/scala/cern/colt/matrix/tfloat/impl/DenseFloatMatrix2DViewTest.scala
package cern.colt.matrix.tfloat.impl

import cern.colt.matrix.MatrixTypes.DenseFloatMatrix2D

class DenseFloatMatrix2DViewTest(arg0: String) extends DenseFloatMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new DenseFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new DenseFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new DenseFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}
